### ISSUE TITLE
Sentence splitter splits sentences in a false manner

### ISSUE THREAD

**Issue by dilanSachi** [ysenarath/sinling#1]
When the content contains  names like **එම්. ඩබ්. ඒ. ඒ. විජේසූරිය**, the sentences are broken in the middle of the names.
As an example when the following content in used as the input,
```
`මධ්‍යම ආරක්‍ෂක සේනා මූලස්ථානය යටතේ සේවයෙන් නියුතු යුද්ධ හමුදා නිලධාරින් සඳහා ‘අන්තර් ජාතික මානූෂිය නීතිය’ පිළිබඳ වැඩමුළුවක් බදාදා (31) වන දින දියතලාව ස්වේච්ඡා බලසේනා පුහුණු පාසල (VFTS) හිදී පැවැත්විය. ඒ සඳහා 112 වන බලසේනාවේ බළසේනාධිපති බ්‍රිගේඩියර් එම්. ඩබ්. ඒ. ඒ. විජේසූරිය සහ 121 වන බලසේනාවේ බළසේනාධිපති බ්‍රිගේඩියර් එන්. සී. සෝමවීර ඇතුළු යුද්ධ හමුදා නිලධාරීන් 20 ක් මෙම වැඩමුළුව සඳහා සහභාගී වුහ. මෙම වැඩමුළුව රතු කුරුස සංවිධානයේ කලාපීය සහ ආරක්‍ෂක හමුදා නියෝජිතයා වන ජැක් ලෙමා මහතා (Mr Jacques Lemay) සහ රතු කුරුස සංවිධානයේ සන්නිවේදන නිලධාරි චන්න ජයවර්ධන යන මහත්වරුන් විසින් මෙහෙයවනු ලැබීය."`

the output is,

['මධ්යම ආරක්ෂක සේනා මූලස්ථානය යටතේ සේවයෙන් නියුතු යුද්ධ හමුදා නිලධාරින් සඳහා ‘අන්තර් ජාතික මානූෂිය නීතිය’ පිළිබඳ වැඩමුළුවක් බදාදා (31) වන දින දියතලාව ස්වේච්ඡා බලසේනා පුහුණු පාසල (VFTS) හිදී පැවැත්විය. ඒ සඳහා 112 වන බලසේනාවේ බළසේනාධිපති බ්රිගේඩියර් එම්. ඩබ්', 'ඒ. ඒ. විජේසූරිය සහ 121 වන බලසේනාවේ බළසේනාධිපති බ්රිගේඩියර් එන්. සී. සෝමවීර ඇතුළු යුද්ධ හමුදා නිලධාරීන් 20 ක් මෙම වැඩමුළුව සඳහා සහභාගී වුහ', 'මෙම වැඩමුළුව රතු කුරුස සංවිධානයේ කලාපීය සහ ආරක්ෂක හමුදා නියෝජිතයා වන ජැක් ලෙමා මහතා (Mr Jacques Lemay) සහ රතු කුරුස සංවිධානයේ සන්නිවේදන නිලධාරි චන්න ජයවර්ධන යන මහත්වරුන් විසින් මෙහෙයවනු ලැබීය.']
```

**Comment by ysenarath** [comment]
Fixed in [commit](https://github.com/ysenarath/sinling/commit/13461c4e513bcb7d67cf4d9de6f4ce7cbe42b25f).

Could you please confirm? Thanks.

**Comment by dilanSachi** [comment]
the issue with the name is fixed now. but the sentence still does not break from **(VFTS) හිදී පැවැත්විය. ඒ සඳහා**
I think the issue is in **tokenizer.py - line 73**
`'පී.', 'කිව්.', 'ආර්.', 'එස්.', 'ටී.', 'ය.', 'ඩබ.', 'ඩබ්ලිව්.', 'ඩබ්.',`
the **ය** in here should be **යූ**

**Comment by ysenarath** [comment]
--Reopened--

Similar Issue-

**Comment by ysenarath** [comment]
Fixed in [#1 (comment)](https://github.com/ysenarath/sinling/issues/1#issue-602397590). Please Check. Thanks.

**Comment by dilanSachi** [comment]
It is fixed now. But now it's missing **වී.**
